### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by hsuehilu** [sharelatex/sharelatex#520]
I would like to request for a feature for page navigation in the built-in PDF viewer 
support the function of "back to the previous view". When I read the generated PDF files, I frequently follow internal PDF links to take a quick look at tables, figures, and references in other pages. After taking a quick look, there does not seem to be an easy way to go back to the original view in the original page.  In Adobe Reader for Mac, the key stroke for this function is COMMAND-LEFTARROW. I use it all the time. Thanks a lot for considering my request.

**Comment by C0D3D3V** [comment]
I would also like such feature.
 This is already integrated in many PDF viewers, e.g. [Okular]( https://github.com/KDE/okular/blob/cb7cd09c82423c5bcf5e9f016edcab4b1ae8ccc5/core/document.cpp#L3771)

Creating a history of the viewport position of the PDF viewer and then being able to jump back and forth at will would make it much easier to navigate in PDFs (especially when clicking on links in PDFs such as in the table of contents or references). 

Or just like [evince](https://gitlab.gnome.org/GNOME/evince/-/blob/main/shell/ev-history.c), which simply remembers which page you were on before you clicked on a link. The same approach uses also [qpdfview](https://github.com/bendikro/qpdfview/blob/8518b4e7654776984294e95c4a5b14f2c6c0867d/sources/documentview.cpp#L1015)

**Comment by lcbarnett** [comment]
I'll add my vote to this feature request - it would be a significant improvement in usability. As others have pointed out, probably most modern PDF viewers (I use mupdf) have this feature.

Meantime, a somewhat clunky workaround - at least if your code location is synched to the current PDF location - is to use the "Go to code location" arrow.

**Comment by wkjarosz** [comment]
I would also like to vote for this, now 8-year-old, feature request.

**Comment by jameshowison** [comment]
Indeed, this is really valuable for numbered citations where you follow the link but then want to get back to the text you were reading.
